wolfssl/test.h

```diff
@@ -1980,7 +1980,7 @@ static INLINE const char* mymktemp(char *tempfn, int len, int num)
 
     typedef struct key_ctx {
         byte name[WOLFSSL_TICKET_NAME_SZ];     /* name for this context */
-        byte key[16];                          /* cipher key */
+        byte key[32];                          /* cipher key */
```

Comment:
That's a good point. Yes and would remove the magic number. Am going to go ahead and force update this to just overwrite the commit since is so small.